### PR TITLE
Added sort to ReadAll

### PR DESCRIPTION
**Pull request by Chromicle**
#### Changes: 
Added sort to ReadAll activity

#### Files changed:
ReadAllData.java
read_all.xml

##### Screenshots of the change: 
ezgif com-video-to-gif

### CODE REVIEW

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java [hunk 1]
```diff
@@ -18,23 +20,26 @@
...
 
 /**
  * Created by Chromicle.
  */
 public class ReadAllData extends AppCompatActivity {
 
+    RadioGroup radioGroup;
+    LinearLayout totalLayout;
+    LinearLayout radioButtonView;
+    int flag;
```
Comment (yashk2000) [inline TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java]: `flag` can be private.

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java [hunk 2]
```diff
@@ -47,11 +52,48 @@ protected void onCreate(Bundle savedInstanceState) {
         listView = (ListView) findViewById(R.id.listView);
         listView.setAdapter(adapter);
         heading = (TextView) findViewById(R.id.heading);
+        radioButtonView = (LinearLayout) findViewById(R.id.radiogroup_view);
```
Comment (yashk2000) [inline TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java]: I think the type casting `LinearLayout` will be redundant. Please remove it.

**Comment by Chromicle** [comment]
> Remove all the redundant type casting on the different views.

I think no redundant view are there,
radioBottonView, totalLayout are used for hiding 
For radioButtomView it is used for hiding when readAll is clicked

**Comment by yashk2000** [comment]
Views are not redundant. The type casting on the views is redundant.

**Comment by yashk2000** [comment]
For example here:
`radioGroup = (RadioGroup) findViewById(R.id.radiogroup);`

The place where you have written `(RadioGroup)` that part is redundant. Open in android studio, it will be marked in a grey color.

**Comment by yashk2000** [comment]
@Chromicle looks good. Please squash the commits.

**Pull request merged**